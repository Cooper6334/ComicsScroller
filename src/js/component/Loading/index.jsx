// @flow
import React, { Component } from 'react';
import cn from './Loading.css';

class Loading extends Component {
  render() {
    return (
      <div className={cn.Loading}>
        <svg>
          <circle cx="30" cy="30" r="25" />
        </svg>
      </div>
    );
  }
}

export default Loading;
