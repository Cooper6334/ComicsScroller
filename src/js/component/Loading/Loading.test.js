import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Loading from './';

test('Loading contains circle svg', () => {
  const LoadingCmp = shallow(<Loading />);
  expect(
    LoadingCmp.contains(
      <svg>
        <circle cx="30" cy="30" r="25" />
      </svg>,
    ),
  ).toBe(true);
});

test('Loading snapshot', () => {
  const tree = renderer.create(
    <Loading />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
