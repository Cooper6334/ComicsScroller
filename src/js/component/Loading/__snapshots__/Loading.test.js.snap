// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading snapshot 1`] = `
<div
  className="Loading"
>
  <svg>
    <circle
      cx="30"
      cy="30"
      r="25"
    />
  </svg>
</div>
`;
